src/components/Sidebar/Sidebar.story.tsx

```diff
@@ -21,37 +27,60 @@ const Header = styled(H)`
   margin-top: 0;
 `;
 
-const Template: Story<Props> = ({ attach = 'left', ...args }) => {
+const Template: Story<Props & { labelAsTooltip?: boolean }> = ({
+  attach = 'left',
+  labelAsTooltip = true,
```

Comment:
You can also remove this and add `labelAsTooltip: { control: 'boolean' }` so designers and engineers can toggle the labels on and off if using the storybook controls. 👍